src/server/pachyderm_test.go

```diff
@@ -6,7 +6,9 @@ import (
 	"compress/gzip"
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"fmt"
+	"github.com/pachyderm/pachyderm/v2/src/debug"
```

Comment:
This isn't a big problem, but your IDE is not doing what gofmt/goimport/gopls would do here (which would be to put this import at line 33).  If it can be adjusted to be consistent with the rest of the codebase, that would be ideal.